pipert2/core/base/pipe.py

```diff
@@ -84,7 +84,7 @@ def link(self, *wires):
         """
 
         for wire in wires:
-            self.wires[wire.source.name] = wire
+            self.wires[f"{wire.source.flow_name}-{wire.source.name}"] = wire
```

Comment:
Let's say we have -
flow1 named - flow 
flow2 named - flow-abc
routine1 in flow1 named - abc-ab
routine2 in flow2 named - ab

Then the wires override themselves. 
The first wire is going to be flow-abc-ab and the second wire will be as same. 
Maybe the key will be a dictionary too in format - 
{ "flow_name": ..., "routine_name": ...}